style.css

```diff
@@ -211,6 +211,12 @@ iframe{
     margin: 0.5rem 0;
```

Comment:
use px of size is less than 1rem

Reply:
0.5rem is equal to 8px & I prefer using it since everywhere I have used rems